Terasology/Dialogs
# new answer button assets

## Description

- An issue with Answer Image positioning:
Even if there was no issue in previous tests, this time it seemed that the positioning was off
- Added two new assets about the answer buttons, one as a default image, and one for when hovering/pushing the button.

## Changes

### For Issue
I believe the issue was that I set a `family` with fixed height at the column hosting the answer images, which family was meant to set a fixed size at the answer UIImages. What I did, was to remove the family from the column, and add it via code to each new UIImage added before the answer UIButtons.

### For background images

#### New Assets
- default one
<img width="200" alt="black" src="https://user-images.githubusercontent.com/48293545/91636211-96915900-ea07-11ea-8361-71a5aa4d9c9f.png">

- hover/push
<img width="200" alt="blackPushed" src="https://user-images.githubusercontent.com/48293545/91636218-a0b35780-ea07-11ea-9ba4-45d687b50b20.png">

#### Results
Because the UIButtons match the width of their column, which matches the width of the wider button (the one with the most text in it), the image seems kind of weird to the buttons with less text. 

The solution should be to set at each UIButtons the LayoutInfo of `"use-content-width": true`, but I am not aware of how to do that via code yet.

<img width="371" alt="Screenshot_4" src="https://user-images.githubusercontent.com/48293545/91636232-b32d9100-ea07-11ea-9bd3-1ad7ef5421ba.png">
<img width="468" alt="Screenshot_5" src="https://user-images.githubusercontent.com/48293545/91636236-b88adb80-ea07-11ea-9f67-58f270f8b543.png">
<img width="312" alt="Screenshot_6" src="https://user-images.githubusercontent.com/48293545/91636237-b9bc0880-ea07-11ea-9e0e-f8ca4d5d05a8.png">

### How to test

1. Start a game which contains a Dialogue box
2. Trigger the dialogue box

If you want to test this with LightAndShadow:
1. Check https://github.com/Terasology/LightAndShadow/pull/156
2. Talk to the NPC (The Fool) in front of you when you start the game
